python/example_code/ec2/scenario_get_started_instances.py

```diff
@@ -1,274 +1,334 @@
+        image_choice = 0
+        console.print(
+            f"- **Simulated input**: Chose AMI: {amzn2_images[image_choice]['ImageId']}\n"
         )
-        print("Great choice!\n")
 
-        print(
+        console.print(
             f"Here are some instance types that support the "
-            f"{amzn2_images[image_choice].architecture} architecture of the image:"
+            f"{amzn2_images[image_choice]['Architecture']} architecture of the image:"
         )
         inst_types = self.inst_wrapper.get_instance_types(
-            amzn2_images[image_choice].architecture
-        )
-        inst_type_choice = q.choose(
-            "Which one do you want to use? ", [it["InstanceType"] for it in inst_types]
+            amzn2_images[image_choice]["Architecture"]
         )
-        print("Another great choice.\n")
-
-        print("Creating your instance and waiting for it to start...")
-        self.inst_wrapper.create(
-            amzn2_images[image_choice],
-            inst_types[inst_type_choice]["InstanceType"],
-            self.key_wrapper.key_pair,
-            [self.sg_wrapper.security_group],
+        inst_type_choice = 0
+        console.print(
+            f"- **Simulated input**: Chose instance type: {inst_types[inst_type_choice]['InstanceType']}\n"
         )
-        print(f"Your instance is ready:\n")
+
+        console.print("Creating your instance and waiting for it to start...")
+        with alive_bar(1, title="Creating Instance") as bar:
+            self.inst_wrapper.create(
+                amzn2_images[image_choice]["ImageId"],
+                inst_types[inst_type_choice]["InstanceType"],
+                self.key_wrapper.key_pair["KeyName"],
+                [self.sg_wrapper.security_group],
+            )
+            time.sleep(21)
+            bar()
+
+        console.print(f"**Success! Your instance is ready:**\n", style="bold green")
         self.inst_wrapper.display()
 
-        print("You can use SSH to connect to your instance.")
-        print(
+        console.print("You can use SSH to connect to your instance.")
+        console.print(
             "If the connection attempt times out, you might have to manually update "
             "the SSH ingress rule for your IP address in the AWS Management Console."
         )
         self._display_ssh_info()
 
     def _display_ssh_info(self):
-        """
-        Displays an SSH connection string that can be used to connect to a running
-        instance.
-        """
-        print("To connect, open another command prompt and run the following command:")
-        if self.eip_wrapper.elastic_ip is None:
-            print(
-                f"\tssh -i {self.key_wrapper.key_file_path} "
-                f"ec2-user@{self.inst_wrapper.instance.public_ip_address}"
-            )
+        console.print(
+            "\nTo connect, open another command prompt and run the following command:",
```

Comment:
This is example code, and we're showing the user how to do something. While I think it's interesting to give them the opportunity to SSH into the instance, I also think this is a very simple task (1 line command) and it's completely sufficient to show them how they would do it without pausing the script.

Reply:
When it ran straight through, I never saw the prompt. The only reason I knew it was there was because I have ran the previous (interactive) version. Otherwise I would have missed it entirely.

Reply:
Added back pause.